src/storage.h

```diff
@@ -164,6 +165,17 @@ class Storage
                       "range": [0, 12000]}
   int residence_time;
 
+  #pragma cyclus var {"default": 0,\
+                      "tooltip":"sell quantity (kg)",\
+                      "doc":"material will be sold in integer multiples of this quantity. If"\
+                      " the buffer contains less than the sell quantity, no material will be"\
+                      " offered", \
```

Comment:
Perhaps this comment should also mention what happens when this variable is 0 (default)

Reply:
Still encourage an expansion of this comment to clarify what happens if the sell quantity is 0

Reply:
```suggestion
                      " offered. When sell_quantity is set to zero, there is no restriction "\
                      " on transaction quantity", \
```